.github/ISSUE_TEMPLATE/dotnetcore_bug.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Bug Report (.NET Core)
```

Comment:
Distinguish between .NET Core and .NET Framework is not really actionable for us.
But its important to distinguish between ASP.NET and ASP.NET Core.

Reply:
I could think the following benefits:
1. Indicating .NET / .NET Core in the issue template name would help to route JS/Python customers away and to the right place.
2. It doesn't make sense to ask people which Platform for .NET as Windows is the only one.
3. We can do auto-tagging for .NET and .NET Core, which helps us to see the trend of people moving from .NET to Core.
4. In this way we no longer need to ask whether it is ASP.NET or ASP.NET Core as it is indicated by the .NET/Core.

Reply:
1. I don't think we need to route JS/Python/Java customers with this template. It should be obvious this repo is not for them. We have never seen customers opening python issue here.

The only time customer misreport issue in this repo is when the artifacts produced by this repo is involved - good examples are Azure Functions, Azure Web Apps code-less attach - both uses the code/artifacts from this repo, but the support/question is NOT provided by this repo. The current template and this PR does not have anything which redirect customers directly to Azure functions for Functions issue - that was one of the key goals of updating this bug template.

2. Agree platform is always windows for .NET. By platform we don't necessarily mean OS. It could mean - are you running in a Azure Web App? Container? SF cluster etc. 

3. Don't think we need GH issues to get that telemetry.

4. As Asp.Net Core apps can be written targeting .NET Framework, we still need to ask this information. 

Knowing which nuget package produced from this repo customer is using - this is the most important information we need -(eg: Its not enough to know if an app is .NET Core or .NET Framework, we need to know if they use basesdk alone, or workerservice or something else)

Reply:
> 2. Agree platform is always windows for .NET. By platform we don't necessarily mean OS. It could mean - are you running in a Azure Web App? Container? SF cluster etc.

We need to be specific, I've changed the wording, please see if it helps to bring clarity.

> 3. Don't think we need GH issues to get that telemetry.

That's okay, I've removed the second template.

> Knowing which nuget package produced from this repo customer is using - this is the most important information we need -(eg: Its not enough to know if an app is .NET Core or .NET Framework, we need to know if they use basesdk alone, or workerservice or something else)

I've made it explicit by asking customers to list the nuget packages, do you think it'll work?